### PR TITLE
Pasteupdate

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- removes the `themeprovider` in favour of using pasteup, which seems to be enough for now
- adds colours and fonts to pasteup

## Why?
it's what we have to do

### CODE REVIEW

File: src/components/Header/Nav/Links/Link.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
     textDecoration: 'none',
+    display: 'none',
     ':hover': {
         textDecoration: 'underline',
     },
     ':focus': {
         textDecoration: 'underline',
     },
-});
+    ...(props.index < 2 && {
```
Comment (sndrs) [inline src/components/Header/Nav/Links/Link.js]: 👋 `nth-child`

Comment (SiAdcock) [reply src/components/Header/Nav/Links/Link.js]: Hmm.... What is this doing?

Comment (SiAdcock) [reply src/components/Header/Nav/Links/Link.js]: Can we call this prop `nthChild`? Would that be more obvious?

Comment (sndrs) [reply src/components/Header/Nav/Links/Link.js]: hmmm, maybe a `hideUntilDesktop` prop?

Comment (SiAdcock) [reply src/components/Header/Nav/Links/Link.js]: Yeah that could work. No big deal though, I think I was just temporarily thrown by the name!

Comment (sndrs) [reply src/components/Header/Nav/Links/Link.js]: like this https://github.com/guardian/guui/pull/38

it's intersting it wasn't clear to you, would be good to alight on something more transparent

**Pull request merged**